website/docs/interactors/3-locators-filters-actions.md

```diff
@@ -78,9 +78,13 @@ The filter object can take as many properties as you want it to:
 TextField({ id: 'username-id', placeholder: 'USERNAME', visible: true }).exists();
 ```
 
-The filters available are defined by each interactor, so look at the API docs for the built-in interactors or the code for your own interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides eight different filters.
+The filters available are defined by each interactor, so look at the source code for the built-in interactors or the code for your own interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters.
 
-The [Assertions](/docs/interactors/assertions) page will cover how you can use filters to assert against Interactors in your tests. And later on the [Writing Interactors](/docs/interactors/write-your-own) page you will learn how to add your own filters to Interactors.
+:::note
+Since the TextField interactor extends on the [FormField interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts) which extends from the [HTML interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts), the TextField interactor inherits all of the FormField and HTML interactor properties. So in total there are actually eleven filters available for the TextField interactor. 
```

Comment:
```suggestion
There are many filters available for the TextField interactor. The TextField interactor inherits all of the [FormField interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts) and [HTML interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts) properties.
```

Once there are more than a few of something, it's best to leave numbers out so that they don't have to be updated when the codebase changes. 

I think it is ok to cut down the explanation some here. If someone doesn't understand inheritance, they would not understand the longer explanation. If they do, then a short explanation should be enough.

Reply:
Sounds good. But what should we do about the paragraph above it?

> `The filters available are defined by each interactor, so look at the source code for the built-in interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters.`

Should we snip out the last sentence starting with `For example`?

Reply:
Here's one way you could make these flow together:

> The filters available are defined by each interactor, so look at the source code for the built-in interactors to know what is available. For example, if you take a look at the [TextField source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/text-field.ts), you'll see that the TextField interactor provides two filters. In addition, TextField inherits all of the properties of the [FormField interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/form-field.ts) and [HTML interactor](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts).